### ISSUE TITLE
Document the default configuration for each module

### ISSUE THREAD

**Issue by raboof** [akka/akka#27798]
We already have all the reference configurations included at https://doc.akka.io/docs/akka/current/general/configuration.html#configuration.

Perhaps it would make it more discoverable to have the reference configuration for each module in a separate file and include it in the TOC structure for that module?

**Comment by patriknw** [comment]
yes, and no, it's also nice to have one single page (with in page browser search) with all settings
